serial/uart.c

```diff
@@ -190,7 +190,9 @@ uart0_rx_intr_handler(void *para)
   if (READ_PERI_REG(UART_INT_RAW(uart_no)) & UART_FRM_ERR_INT_RAW) {
     uint32 now = system_get_time();
     if (last_frm_err == 0 || (now - last_frm_err) > one_sec) {
+#ifdef UART_DBG
       os_printf("UART framing error (bad baud rate?)\n");
```

Comment:
Right, I was indiscriminate with the ifdef on all the os_printf because I
was trying to reduce flash size
On Sep 12, 2015 12:22 AM, "Thorsten von Eicken" <notifications@github.com>
wrote:

> In serial/uart.c
> <https://github.com/jeelabs/esp-link/pull/47#discussion_r39334395>:
>
> > @@ -190,7 +190,9 @@ uart0_rx_intr_handler(void *para)
> >    if (READ_PERI_REG(UART_INT_RAW(uart_no)) & UART_FRM_ERR_INT_RAW) {
> >      uint32 now = system_get_time();
> >      if (last_frm_err == 0 || (now - last_frm_err) > one_sec) {
> > +#ifdef UART_DBG
> >        os_printf("UART framing error (bad baud rate?)\n");
>
> This is an error printf, not a debug printf
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeelabs/esp-link/pull/47/files#r39334395>.
>